### PR TITLE
Add version: TortoiseSVN.TortoiseSVN version 1.14.6.29673

### PR DESCRIPTION
**Pull request by PckgrBot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132147)

### CODE REVIEW

File: manifests/t/TortoiseSVN/TortoiseSVN/1.14.6.29673/TortoiseSVN.TortoiseSVN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Upgrade: /norestart
+  Custom: /norestart
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://master.dl.sourceforge.net/project/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.6.29673-win32-svn-1.14.3.msi/download
+  InstallerSha256: 8CF5CE27D21490C24EEDF91E0AC2BC4A748BA8F4EB20CB7C1FC9442D2D580008
```
Comment (fancybody) [inline manifests/t/TortoiseSVN/TortoiseSVN/1.14.6.29673/TortoiseSVN.TortoiseSVN.installer.yaml:20]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/tortoisesvn/files/1.14.6/Application/TortoiseSVN-1.14.6.29673-win32-svn-1.14.3.msi
  InstallerSha256: 7b1e6717142dc8d3c5ecf871a18377795d66d0c4546ee20f18c8781ef537a26a
```

File: manifests/t/TortoiseSVN/TortoiseSVN/1.14.6.29673/TortoiseSVN.TortoiseSVN.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  UnsupportedOSArchitectures:
+  - x64
+  - arm64
+- Architecture: x64
+  InstallerUrl: https://master.dl.sourceforge.net/project/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.6.29673-x64-svn-1.14.3.msi/download
+  InstallerSha256: 8CF5CE27D21490C24EEDF91E0AC2BC4A748BA8F4EB20CB7C1FC9442D2D580008
```
Comment (fancybody) [inline manifests/t/TortoiseSVN/TortoiseSVN/1.14.6.29673/TortoiseSVN.TortoiseSVN.installer.yaml:26]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/tortoisesvn/files/1.14.6/Application/TortoiseSVN-1.14.6.29673-x64-svn-1.14.3.msi
  InstallerSha256: 6597da34055c3a8ea6a91caef2b9ea24474b93fef60c4b123f3fc5ed7b6faf6b
```

File: manifests/t/TortoiseSVN/TortoiseSVN/1.14.6.29673/TortoiseSVN.TortoiseSVN.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  ProductCode: '{E6F481E7-EF30-4811-A73C-1379B2C94595}'
+  UnsupportedOSArchitectures:
+  - x86
+  - arm64
+- Architecture: arm64
+  InstallerUrl: https://master.dl.sourceforge.net/project/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.6.29673-ARM64-svn-1.14.3.msi/download
+  InstallerSha256: 8CF5CE27D21490C24EEDF91E0AC2BC4A748BA8F4EB20CB7C1FC9442D2D580008
```
Comment (fancybody) [inline manifests/t/TortoiseSVN/TortoiseSVN/1.14.6.29673/TortoiseSVN.TortoiseSVN.installer.yaml:33]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/tortoisesvn/files/1.14.6/Application/TortoiseSVN-1.14.6.29673-ARM64-svn-1.14.3.msi
  InstallerSha256: 1769a5523edca7342bc1b24d80f38c40c4901925affca12ffef11e0125810f74
```

**Review by fancybody — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Hi @PckgrBot,

> This PR's version number 1.14.6.29673 has 4 parameters (sets of numbers between dots - major, minor, etc), which is greater than the current manifest's version 1.14.29465, which has 3 parameters.

Is this intentional?

(Automated response - build 543)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**